apps/rstudio.talon

```diff
@@ -0,0 +1,126 @@
+app: RStudio
```

Comment:
Would you mind a few tweaks? Since these look like mac-specific shortcuts, I'd propose two minor tweaks:
-Add os: mac
-moving this into apps\mac
 
Trying to stick to the convention where os-specific stuff goes into the os-specific folder, until we have cross-platform plugins or whatever.